components/tag/theme.css

```diff
@@ -0,0 +1,114 @@
+@import '@teamleader/ui-colors';
+@import '@teamleader/ui-typography';
+@import '@teamleader/ui-utilities';
+
+:root {
+  --tag-border-radius: calc(.4 * var(--unit));
+  --tag-border-width: 2px;
+  --tag-size-small: 24px;
+  --tag-size-medium: 30px;
+  --tag-size-large: 36px;
+}
+
+.tag {
+  composes: reset;
+  background-color: color(var(--color-neutral-darkest) a(12%));
+  border-radius: var(--tag-border-radius);
+  display: inline-flex;
+  align-items: center;
+}
+
+.label {
+  color: var(--color-teal-darkest);
+  font-family: var(--font-family-medium);
+}
+
+.remove-button {
+  border-radius: 0 var(--tag-border-radius) var(--tag-border-radius) 0;
+  height: 100%;
+}
+
+.label-button {
+  border: 0 !important;
```

Comment:
no other way to solve this without using `!important?`

Reply:
Not realy, but I think it's allright to use `!important` for a specific css property override. Especially because we work with css-modules, so the override is isolated inside it's own component and won't affect others.

 :ok: by @driesd on 13/11/2017 16:05:25